js/stronghold.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+// ----------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+
+// ----------------------------------------------------------------------------
+
+module.exports = class Stronghold extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'stronghold',
+            'name': 'Stronghold',
+            'country': ['US'],
+            'rateLimit': 1000,
+            'version': 'v1',
+            'comment': 'This comment is optional',
+            'urls': {
+                'logo': 'https://example.com/image.jpg',
+                'api': {
+                    'public': 'https://api.stronghold.co',
+                    'private': 'https://api.stronghold.co',
+                },
+                'www': 'https://www.example.com',
+                'doc': [
+                    'https://docs.stronghold.co/',
+                ],
+            },
+            'venueId': 'trade-public',
```

Comment:
let's move this to `options`:

```
...
'options': {
    // exchange-specific controls
    'venueId': 'trade-public',
},
...
```

↑ Otherwise Python and PHP will complain upon trying to set `self.venueId` which isn't declared anywhere as it isn't standard.